<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesDeleteWindow</class>
 <widget class="QDialog" name="NotesDeleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="DeleteNote">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>670</width>
     <height>320</height>
    </rect>
   </property>
   <property name="title">
    <string>Delete Note</string>
   </property>
   <widget class="QGroupBox" name="TasksBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>310</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Tasks</string>
    </property>
    <widget class="QTableView" name="TasksTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>290</width>
       <height>190</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="NotesForTheTaskBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>310</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>Notes For The Task</string>
    </property>
    <widget class="QTableView" name="NotesForTheTaskTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>290</width>
       <height>190</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>630</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteNoteButton">
       <property name="text">
        <string>Delete Note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
